docs/docs/cmd/aad/app/app-remove.md

```diff
@@ -1,9 +1,15 @@
-# aad app delete
+# aad app remove
```

Comment:
After renaming the file, we should update the link to it in release notes. Right now we're getting an error when building docs